subread/2.0.0/featureCounts.nf

```diff
@@ -0,0 +1,30 @@
+process featureCounts {
```

Comment:
'Use CamelCase for tool, command and process names'

Reply:
Fixed